dfsclient/src/main/java/org/hpccsystems/dfs/client/HPCCRemoteFileWriter.java

```diff
@@ -85,6 +87,12 @@ public HPCCRemoteFileWriter(DataPartition dp, FieldDef recordDef, IRecordAccesso
 
         this.binaryRecordWriter = new BinaryRecordWriter(this.outputStream);
         this.binaryRecordWriter.initialize(this.recordAccessor);
+
+        log.info("HPCCRemoteFileWriter: Opening file part: " + dataPartition.getThisPart()
+                + " compression: " + fileCompression.name() + " "
+                + " record definition:\n"
+                + RecordDefinitionTranslator.toJsonRecord(this.recordDef));
```

Comment:
What's the bloat factor of this output?
Was there discussion of reserving this output for error/warning situations only?